src/triage/cli.py

```diff
@@ -383,29 +383,37 @@ def __call__(self, args):
 class Db(Command):
     """Manage experiment database"""
 
-    @cmdmethod
+    @cmdmethod("revision", help="database schema revision to upgrade to (see triage db history)")
     def upgrade(self, args):
         """Upgrade triage results database"""
-        upgrade_db(dburl=self.root.db_url)
+        upgrade_db(revision=args.revision, dburl=self.root.db_url)
+
+    @cmdmethod("revision", help="database schema revision to downgrade to (see triage db history)")
+    def downgrade(self, args):
+        """Downgrade triage results database"""
+        downgrade_db(revision=args.revision, dburl=self.root.db_url)
 
-    @cmdmethod(
-        "configversion",
-        choices=REVISION_MAPPING.keys(),
-        help="config version of last experiment you ran",
-    )
+    @cmdmethod("revision", help="database schema revision to stamp to (see triage db history)")
     def stamp(self, args):
-        """Instruct the triage results database to mark itself as updated to a
-        known version without doing any upgrading.
+        """Mark triage results database as updated to a known version without doing any upgrading.
+
+        The revision can be anything alembic recognizes, such as a specific revision or 'head' (the most recent revision in the current codebase)
+
+        This is most useful if the database was created without a 'results_schema_versions' table (i.e. old versions of triage that didn't enforce alembic use), but could also be useful after general database mangling.
 
-        Use this if the database was created without an 'alembic_version' table.
-        Uses the config version of your experiment to infer what database version is suitable.
+        If you don't know what the right revision is, here are some database revisions that old experiment configs are associated with:
+            - no config version: 8b3f167d0418
+            - v1 or v2: 72ac5cbdca05
+            - v3: 7d57d1cf3429
+            - v4: 89a8ce240bae
+            - v5: 2446a931de7a
```

Comment:
just FYI, `argparse` is a supreme jerk about description text, and might mangle this beyond utility -- might be worth checking it out, (just run `triage db stamp --help`).

it is _possible_ to force it to respect your whitespace, but I don't remember how offhand, and IIRC it's a bit involved.

Regardless, as much as I like this here, it _might_ make more sense in documentation elsewhere.

Reply:
Yeah it doesn't look great. I don't think creating a CLI doc is in the scope of this PR, but I might make that separately.